content/docs/introduction/monitoring-page.md

```diff
@@ -120,6 +120,10 @@ The **Rows** graph shows the number of rows deleted, updated, and inserted over
 
 Tracking rows inserted, updated, and deleted over time provides insights into your database's activity patterns. You can use this data to identify trends or irregularities, such as insert spikes or an unusual number of deletions.
 
+<Admonition type="note">
+This graph excludes data for rows deleted by `TRUNCATE TABLE` operations. It only shows data for row-level operations.
```

Comment:
Suggestion: Row metrics only capture (or a better synonym) row-level changes (INSERT, UPDATE, DELETE, etc.) and exclude table-level operations such as TRUNCATE.